packages/web3-wrapper/src/index.ts

```diff
@@ -74,37 +64,22 @@ export class Web3Wrapper {
     public getNetworkId(): number {
         return this.networkId;
     }
-    public async getContractInstanceFromArtifactAsync(
-        artifact: Artifact, address?: string,
-    ): Promise<Web3.ContractInstance> {
-        let contractAddress: string;
-        if (_.isUndefined(address)) {
-            const networkId = this.getNetworkId();
-            if (_.isUndefined(artifact.networks[networkId])) {
-                throw new Error(ZeroExError.ContractNotDeployedOnNetwork);
-            }
-            contractAddress = artifact.networks[networkId].address.toLowerCase();
-        } else {
-            contractAddress = address;
-        }
-        const doesContractExist = await this.doesContractExistAtAddressAsync(contractAddress);
-        if (!doesContractExist) {
-            throw new Error(CONTRACT_NAME_TO_NOT_FOUND_ERROR[artifact.contract_name]);
-        }
-        const contractInstance = this.getContractInstance(
-            artifact.abi, contractAddress,
-        );
-        return contractInstance;
-    }
     public toWei(ethAmount: BigNumber): BigNumber {
         const balanceWei = this.web3.toWei(ethAmount, 'ether');
         return balanceWei;
     }
     public async getBalanceInWeiAsync(owner: string): Promise<BigNumber> {
         let balanceInWei = await promisify<BigNumber>(this.web3.eth.getBalance)(owner);
+        // Rewrap in a new BigNumber
         balanceInWei = new BigNumber(balanceInWei);
         return balanceInWei;
     }
+    public async getBalanceInEthAsync(owner: string): Promise<BigNumber> {
```

Comment:
Um, this was probably taken from `website`. It should be removed. I'm not a fan of this method and feel it should be refactored to return wei, not ETH.